### ISSUE TITLE
Excavator sound while firing at enemies.

### ISSUE THREAD

**Issue by Matty4818** [pa1nki113r/Project_Brutality#649]
Please forgive me if this was an issue that was brought up already. I just couldn’t look through all the issues to find out about it.  When firing the excavator in drop mode at a large group of enemies, the sound cuts off right after firing. If you aim away from the enemies and fire, it works fine. I’ve tried both the PB3.0 Master Zip file, along with the PB3.0 Jmartinez branch file and they both do the same. Just thought that I would let you guys know about it. Thank you! -MattMayhem

**Comment by JMartinez9820** [comment]
Excavator is subjected to be overhauled in the future, although it won't happen until later. I may consider changing how the sound works, since it uses the extremely outdated A_PlayWeaponSound, rather than the more updated A_PlaySound or A_StartSound.

**Comment by JMartinez9820** [comment]
Although this is not a priority at the moment.

**Comment by Matty4818** [comment]
Thank you very much. I appreciate the fast response. If it’s not too much trouble please let me know when the updated/fixed version comes out!

**Comment by JMartinez9820** [comment]
@Matty4818 https://github.com/JMartinez9820/Project_Brutality/tree/JMartinezBranch
See if my branch has fixed it, I changed the firing sound channel to zero, which means it automatically select the channel for that sound.

**Comment by Matty4818** [comment]
I pinpointed the problem I believe. When I use the Glory_Kill_v24h.pk3 from the Discord Server, the Excavator sound cuts off when I'm firing. Is there any way that can be fixed? I used your current version without any mods and then I started adding them one by one to see if they were causing the problem and sure enough when I added the Glory Kills, the sound started cutting off.


-----Original Message-----
